build.gradle

```diff
@@ -129,6 +133,23 @@ dependencies {
     implementation files(new File(buildDir, 'server')) {
         builtBy 'copyDeps'
     }
+    intellijPlatform {
+        // For a full list of IntelliJ IDEA releases please see https://www.jetbrains.com/intellij-repository/releases
+        create(providers.gradleProperty("platformType"), providers.gradleProperty("platformVersion"))
+        bundledPlugins providers.gradleProperty("platformBundledPlugins").orElse("").get().split(',').toList()
+
+        def lsp4ij = providers.gradleProperty('useLocal') && providers.gradleProperty('useLocal') == 'true' ?
+                file("../lsp4ij/build/distributions/LSP4IJ/") :
+                'com.redhat.devtools.lsp4ij:' + lsp4ijVersion
+        plugins lsp4ij
+        pluginVerifier()
+        zipSigner()
+        instrumentationTools()
```

Comment:
Could you explain why these lines needed to be added?

Reply:
Added a comment - https://github.com/OpenLiberty/liberty-tools-intellij/issues/237#issuecomment-2481694429 - to address whether or not we need `zipSigner()` in the future.